packages/mds-db/migration.ts

```diff
@@ -130,11 +134,19 @@ async function doMigration(exec: SqlExecuterFunction, migration: MIGRATION, migr
   }
 }
 
+async function alterGeographiesColumns(exec: SqlExecuterFunction) {
+  await exec(`ALTER TABLE ${schema.TABLE.geographies} RENAME COLUMN previous_geography_ids TO prev_geographies`)
+  await exec(`ALTER TABLE ${schema.TABLE.geographies} ADD COLUMN publish_date bigint`)
+  await exec(`ALTER TABLE ${schema.TABLE.geographies} ADD COLUMN effective_date bigint`)
+  await exec(`ALTER TABLE ${schema.TABLE.geographies} ADD COLUMN description varchar(255)`)
```

Comment:
There are constants defined in the schema for the new column names and types that should be used to construct the `ALTER TABLE` commands.